components/popover/stories/popover.stories.js

```diff
@@ -188,14 +202,153 @@ Nested.args = {
 		}),
 	],
 };
+Nested.tags = ["!dev"];
+
+/**
+ * Popovers can display different elements within their content area. This example uses the
+ * [dialog](?path=/docs/components-dialog--docs) component within the popover.
+ */
+export const DialogStyle = Template.bind({});
+DialogStyle.storyName = "Dialog style content";
+DialogStyle.args = {
+	withTip: true,
+	isOpen: true,
+	trigger: () => null,
+	content: [
+		() => Dialog({
+			showModal: false,
+			// @todo replace custom class with 'size: "small"' arg when added Dialog control is merged.
+			customClasses: ["spectrum-Dialog--small"],
+			isDismissable: false,
+			heading: "Example heading",
+			content: [
```

Comment:
Might be a todo for later but it seems like the Dialog content should automatically wrap in the Typography component. If we push it through the `renderContent` utility exported from `@spectrum-css/preview/decorators`? We might need to update that function to use Typography styles if the content entry is a string?

Reply:
I'm not sure if this completely still applies after the latest set of changes. Let me know if you think there is something that needs updating for this PR.